### DISCUSSION TITLE
Support of HTTP Basic Auth

### DISCUSSION THREAD

**Discussion by Diavolezza** [sindresorhus/Plash#118] [Q&A] [answered]
I want to display a website that uses HTTP Basic Auth. That means when opening that page in a browser a popup-window ist shown. Ist this supported by Plash? 
It seems, that the window is not shown and therefore the website ist not displayed. Or am I doing something wrong?

**Comment by sindresorhus** [comment, marked as answer]
Plash supports that. You can try by adding https://jigsaw.w3.org/HTTP/Basic/ to Plash.
